bundles/org.eclipse.ui.editors/src/org/eclipse/ui/texteditor/AbstractDecoratedTextEditorPreferenceConstants.java

```diff
@@ -220,6 +220,30 @@ private AbstractDecoratedTextEditorPreferenceConstants() {
 	 * </p>
 	 */
 	public final static String EDITOR_LINE_NUMBER_RULER= "lineNumberRuler"; //$NON-NLS-1$
+	
+	/**
+	 * A named preference that controls whether the modern "Find/Replace"-Overlay is used in place
+	 * of the classical Find/Replace-Dialog.
```

Comment:
Please avoid the word "modern". And some more proposal to improve the description:
```suggestion
	 * A named preference that controls whether find and replace functionality in an editor is accessed via an overlay within the editor instead of a classical dialog.
```